packages/ajax/README.md

```diff
@@ -187,6 +187,157 @@ export const cancelConcurrentRequests = () => html`
 `;
 ```
 
+## Ajax Cache
+
+A caching library that uses `lion-web/ajax` and adds cache interceptors to provide caching for use in
+frontend `services`.
+
+> Technical documentation and decisions can be found in
+> [./docs/technical-docs.md](./docs/technical-docs.md)
+
+### Getting started
+
+Consume the global `ajax` instance and add the interceptors to it, using a cache configuration
+which is applied on application level. If a developer wants to add specifics to cache behavior
+he or she has to provide a cache config per action (get etc) via `lionCacheOptions` field of local ajax config,
+see examples below.
+
+> **Note**: make sure to add the **interceptors** only **once**. This is usually
+> done on app-level
+
+```js
+import {
+  ajax,
+  lionCacheRequestInterceptorFactory,
+  lionCacheResponseInterceptorFactory,
+} from '@lion-web/ajax.js';
+
+const globalCacheOptions = {
+  useCache: 'always',
+  timeToLive: 1000 * 60 * 5, // 5 minutes
+};
+const getCacheIdentifier = () => getActiveProfile().refreshToken;
```

Comment:
This is probably very application specific. If we do show the profile, we shouldn't use the `refreshToken` because it will change each refresh. We want to use the profileId here.

Reply:
thanks @LarsDenBakker